app/serializers/api/questionnaire_serializer.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+module Api
+  class QuestionnaireSerializer < ActiveModel::Serializer
+    attributes :title,
+               :key,
+               :name,
+               :content
```

Comment:
We zouden hier eigenlijk nog een attribute mee moeten geven die een hash is van variable substitutions. We exporteren nu namelijk allemaal teksten met {{deze_student}} en dergelijke. Of vind je dat wij die substitutie zelf al moeten doen? Dwz dat wij overal al deze_student zijn_haar_student etc vervangen en dat de frontend dat niet meer hoeft te doen?

Eigenlijk kan het serverside vervangen hier niet, want dit is een generieke questionnaire API voor een vragenlijst die niet bij een response van een student of begeleider hoort, maar de vraag is of we dat wel willen. Want is deze API alleen even iets om een JSON van iets dat op een vragenlijst op te leveren te testen? Of gaat dit later echt gebruikt worden in de mobiele app? 

Dat is wel een impactvolle beslissing denk ik dus we moeten even denk waar we die verantwoordelijkheid willen hebben als we later een gesplitste frontend en backend hebben. Ik zou zeggen van niet, dwz, dat we later eigenlijk gewoon precies hetzelfde questionnaire fill-out url kunnen aanroepen, maar dan met .json erachter, en dat de backend dan iets oplevert waar alle variabelen al in zijn vervangen. Als de vragenlijst namelijk in de app gecached wordt, dan kan je krijgen dat als we wijzigingen aan vragenlijsten doorvoeren, dat die dan niet direct bij iedereen te zien zijn, tenzij we daar weer iets slims voor bedenken dat een expire cache signaal aan iedereen stuurt oid. Maar je zou dan ook kunnen denken, als je de variabelen vervangt, waarom dan bijv niet direct ook de `show_after` questions? Dat je dingen met een show_after die niet meer geldig is gewoon niet laat meestuurt in de json? Misschien willen we dit wel helemaal niet en willen we juist al die logica naar de frontend verplaatsen, maar hoe zit dat dan als we verschillende frontends hebben, moeten die dan al die show_after expiration/variabele substitution logica kopiëren en in sync houden? Of maken we hier iets van een microservice voor die er tussen hangt en dat soort zaken doet?

Hier moeten we het nog maar een keer over hebben.